### ISSUE TITLE
Using CSS to fingerprint

### ISSUE THREAD

**Issue by arielperez82** [Valve/fingerprintjs2#124]
See: https://github.com/mmulazzani/SHPF/tree/master/Features/CSSFingerprint

Thinking CSS fingerprinting could add another valuable vector.

**Comment by Valve** [comment]
Thanks! Will research this

**Comment by akenn** [comment]
I took a look at that example above and it looks like they create an array of CSS properties that are newer/have mixed adoption. They then test if the browser supports those properties by [setting each property as an inline style](https://github.com/mmulazzani/SHPF/blob/8f7ee6f073cc26fd92b41116cd9e7b989a8ff7ee/Features/CSSFingerprint/CSSFingerprintChecker.php#L506) on an element and [checking the `.style` attribute for the existence of that property](https://github.com/mmulazzani/SHPF/blob/8f7ee6f073cc26fd92b41116cd9e7b989a8ff7ee/Features/CSSFingerprint/CSSFingerprintChecker.php#L510). If that property is undefined or equal to an empty string, the browser doesn't support it. Pretty neat.

I made a quick demo to demonstrate what they're doing using `-webkit-backdrop-filter` (which is only supported only in Safari right now): http://jsbin.com/yinoqizope/edit?html,js,output

I think this would be trivial to implement with just Javascript. Would it be worth adding this test to fingerprintjs?
